BaaServiceKitTests/ProofTests/ProofModelTests.swift

```diff
@@ -0,0 +1,47 @@
+import XCTest
+@testable import BaaServiceKit
+
+class ProofModelTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    func testProofResponse() throws {
+        let file = Bundle(for: type(of: self)).url(forResource: "PartialFakeProofResponse", withExtension: "json")!
+        
+        do {
+            let data = try Data(contentsOf: file)
+            let proofResponse = try ChainpointProofResponse.jsonDecoder.decode([ChainpointProofResponse].self, from: data)
+            
+            // Proof reponse root object tests
+            XCTAssertEqual(proofResponse.count, 1)
+            XCTAssertEqual(proofResponse.first!.hashIdNode, "a4b52f60-7322-11e8-876e-0159403461ed")
```

Comment:
instead of force unwrapping, you could just use the subscript like `proofResponse[0]`